src/sonic-host-services/scripts/hostcfgd

```diff
@@ -793,6 +793,12 @@ class HostConfigDaemon:
                 syslog.syslog(syslog.LOG_WARNING, "Enable state of feature '{}' is None".format(feature_name))
                 continue
 
+            # dhcp_relay WA for production issue
+            if feature_name == 'dhcp_relay':
+                device_metadata = self.config_db.get_table('DEVICE_METADATA')
+                if not (device_metadata and device_metadata['localhost'] and device_metadata['localhost']['type'] and device_metadata['localhost']['type'] != "ToRRouter"):
```

Comment:
Please move this to state field in the init_cfg.json similar to `mux` feature in the test cases.

Reply:
I tried to do so on init_cfg.json but this is not possible since this file is used on build stage and the value of "device_metadata['localhost']['type']" is not exist, so the approach here is to disable dhcp_relay by default.
hostcfgd will check on runtime if this is a ToRRouter switch and enable it accordingly.
If not, it will be disabled as desired, but if the user explicitly request to enable it, this will be accepted as well.

Reply:
The content of the state will be evaluated when loading the init_cfg.j2 for the first time. Also, the build J2 template will not render contents inside `append` call. Something like the following:

```
{% do features.append(("dhcp_relay", "{% if 'type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'ToRRouter' %}enabled{% else %}disabled{% endif %}", false, "enabled")) %}
```

Then we can remove the change in `hostcfgd` all together.

Reply:
@tahmed-dev can you please check the changes now?